pkg/build/builder/util_linux.go

```diff
@@ -16,17 +16,30 @@ func GetCGroupLimits() (*s2iapi.CGroupLimits, error) {
 	// for list of cgroupv2 files to try, but Nalin relayed that examination of the crun and runc code that 'memory.high'
 	// is not used.
 	file := "/sys/fs/cgroup/memory/memory.limit_in_bytes"
-	if cgroups.IsCgroup2UnifiedMode() {
+	cgroupV2 := cgroups.IsCgroup2UnifiedMode()
+	if cgroupV2 {
 		file = "/sys/fs/cgroup/memory.max"
 	}
 	byteLimit, err := readMaxStringOrInt64(file)
 
 	if err != nil {
+		_, e := os.Stat("/sys/fs/cgroup")
+		notExist := false
+		if e != nil && os.IsNotExist(e) {
```

Comment:
new code is supposed to use `errors.Is(e, os.ErrNotExist)` but it does not really matter here.